packages/trail-fastify-plugin/lib/schemas/trails.js

```diff
@@ -105,21 +121,15 @@ const trailSchema = {
     .prop('where', S.object().description('Trail where'))
     .prop('why', S.object().description('Trail reason'))
     .prop('meta', S.object().description('Trail meta')),
-  response: S.object()
-    .description('A audit trail')
-    .additionalProperties(false)
-    .raw({ meta: { id: 'models/trail.response' } })
-    .prop('id', S.number()
-      .description('Trail id')
-      .examples([12345]))
-    .prop('when', dateTime)
-    .prop('who', namedObject('Trail actor'))
-    .prop('what', namedObject('Trail subject'))
-    .prop('subject', namedObject('Trail target'))
-    .prop('where', S.object().additionalProperties(true).description('Trail where'))
-    .prop('why', S.object().additionalProperties(true).description('Trail reason'))
-    .prop('meta', S.object().additionalProperties(true).description('Trail meta'))
-    .required(['when', 'who', 'what', 'subject'])
+  response: responseObject,
+  searchResponse: S.object()
```

Comment:
missing .description() at the top level object

Reply:
will add the same